content/docs-beta/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Documentation - Beta"
+linkTitle: "Docs - SUSHI 1.0.0-beta.1"
```

Comment:
We might go through a few betas without needing to update the docs.  So let's avoid specific reference to `beta.1`.  How about `Docs: SUSHI 1.0 BETA` or `Docs for SUSHI 1.0 BETA` (or `Beta` if you prefer)?